core/nodejs8Action/build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'eclipse'
+eclipse {
+    project {
+        natures 'org.eclipse.wst.jsdt.core.jsNature'
+        buildCommand 'org.eclipse.wst.jsdt.core.javascriptValidator'
+    }
+}
+
+ext.dockerImageName = 'nodejs8action'
+apply from: '../../gradle/docker.gradle'
+distDocker.dependsOn 'copyProxy'
+distDocker.finalizedBy('rmProxy')
+
+task copyProxy(type: Copy) {
+    from '../nodejsActionBase'
```

Comment:
Good feedback, I will change it.

I took the example from swift3 runner
https://github.com/openwhisk/openwhisk/blob/master/core/swift3Action/build.gradle